src/main/webapp/resources/js/components/project/ProjectNav.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { render } from "react-dom";
+import { Layout, Menu, PageHeader, Tag, Tooltip } from "antd";
+import { setBaseUrl } from "../../utilities/url-utilities";
+import { IconFolder, IconSwap } from "../icons/Icons";
+import { blue6, red6 } from "../../styles/colors";
+import { RemoteProjectStatus } from "./RemoteProjectStatus";
+
+const { Item } = Menu;
+const { Content } = Layout;
+
+export function ProjectNav() {
+  /*
+  Get the current page from the global project object
+   */
+  const [current, setCurrent] = useState(window.project.page);
+
+  return (
+    <PageHeader
+      title={window.project.label}
+      avatar={{ icon: <IconFolder /> }}
+      tags={[<RemoteProjectStatus key="remote" />].filter((f) => f !== null)}
+    >
+      <Content>
+        <Menu mode="horizontal" selectedKeys={[current]}>
+          <Item key="samples">
+            <a href={setBaseUrl(`projects/${window.project.id}/samples`)}>
```

Comment:
Set `window.project.id` as a constant as it is used multiple times

Reply:
Updated in 1fbb0cd49c9077eb611714fd1e0d1fbfa9525180